### PR TITLE
LPS-195831 Update Elasticsearch 7 to 7.17.14 (Client & Sidecar)

### PR DESCRIPTION
**Pull request by xbrianlee**

### CODE REVIEW

**Comment by xbrianlee** [comment]
ci:test:search

**Comment by xbrianlee** [comment]
ci:test:search-remote-minimum

**Comment by xbrianlee** [comment]
ci:test:search

**Comment by xbrianlee** [comment]
ci:test:search-remote-minimum

**Comment by AlmirFe** [comment]
Hi @xbrianlee, this issue may be caused because the dependency has not yet been added to Nexus. See [LPS-199517](https://liferay.atlassian.net/browse/LPS-199517)

**Comment by BryanEngler** [comment]
@xbrianlee LPS-199517 is complete

File: test.properties [hunk 1]
```diff
@@ -8366,7 +8366,7 @@
     #
 
     search.engine[search-remote-minimum]=remote-elasticsearch
-    suite.elastic.version[search-remote-minimum]=7.16.3
+    suite.elastic.version[search-remote-minimum]=7.17.0
```
Comment (BryanEngler) [inline test.properties:8369]: @xbrianlee Are we sure about this bump?
on https://liferay.atlassian.net/browse/LPS-195831 under Related-tasks (QA / Engineering)
see point 3: ci:search-remote-minimum should continue be using 7.16.3 to match the client's version

**Comment by xbrianlee** [comment]
ci:test:search

**Comment by xbrianlee** [comment]
ci:test:search

Re-running since CI Infrastructure was restarted

**Comment by xbrianlee** [comment]
ci:test:search:nocompile

Re-running since CI Infrastructure was restarted again

**Comment by xbrianlee** [comment]
@AlmirFe It seems like Sidecar and the Client aren't on 7.17.14 still. I verified this locally as well. Are there still other changes needed?
image

File: test.properties (hunk 1 above)
Comment (BryanEngler) [reply test.properties:8369]: ya it looks good, i think the rule is, if you bump the client, you have to bump the server too.

Comment (BryanEngler) [reply test.properties:8369]: at least for "guaranteed" compatibility

Comment (BryanEngler) [reply test.properties:8369]: although i thought thats why we have the minimum suite, to test if things still work with the "unguaranteed" situation? @lipusz

**Comment by BryanEngler** [comment]
thats interesting... in my local testing everything was updated. theres a few additional commits on https://github.com/liferay-search/liferay-portal/pull/1128 if you want to check that out

File: test.properties (hunk 1 above)
Comment (lipusz) [reply test.properties:8369]: We can bump the minimum suite also to 7.17. Elasticsearch does not "support" connecting a newer client to an older minor version server anyways.

**Pull request closed without merging**